tests/integration/acp.go

```diff
@@ -88,7 +88,7 @@ type AddPolicy struct {
 	Policy string
 
 	// The policy creator identity, i.e. actor creating the policy.
-	Identity immutable.Option[identityRef]
```

Comment:
though: this is a bit strange after being persuaded during the code review or standup discussion into making it `Option` seeing it gone again here. I'm curious to know how it is different now @shahzadlone